lib/matplotlib/tests/test_colors.py

```diff
@@ -526,6 +526,27 @@ def test_Normalize():
     assert 0 < norm(1 + 50 * eps) < 1
 
 
+def test_FuncNorm():
+    def _forward(x):
```

Comment:
```suggestion
    def forward(x):
```
These functions are in the local scope of the `test_FuncNorm()` function and therefore don't need to be marked as "private" by prefixing with an underscore. 

Same with following `_forward` / `_inverse` functions.

Reply:
Sure, but a user should probably make these private if they just cut and paste code, or at least that was my thinking.

Reply:
That may depends on the context.

Anyway, I don't consider tests as cut and paste code. In particular not in the sense that is should fulfill style requirements of the paste-target. I'd rather have the test code styled appropriately in its own context.

However, I don't care to much, so change it or leave it as you see fit.